docs/6.x.yaml

```diff
@@ -1,43 +1,55 @@
 site_name: Gravitational Gravity
-site_url: https://gravitational.com/gravity/docs/ver/6.x/
+site_url: https://gravitational.com/gravity/docs
 repo_url: https://github.com/gravitational/gravity
 site_description: Gravity Documentation
 site_author: Gravitational Inc
 copyright: Gravitational Inc, 2017-19
 
-theme: readthedocs
-theme_dir: theme
-
 # UPDATE HERE FOR EACH VERSION
 site_dir: ../build/docs/6.x
 docs_dir: "6.x"
 
+theme:
+    name: null
+    custom_dir: 'theme/'
+    include_search_page: true
 markdown_extensions:
-  - markdown_include.include
+  - markdown_include.include:
+      base_path: '../'
   - admonition
   - def_list
   - footnotes
-
+  - toc:
+      marker: '[TOC]'
 extra_css: []
+plugins:
+    - search:
+        separator: '[\s\-\.]'
+    - markdownextradata: {
+    }
 extra_javascript: []
 extra:
     version: 6.x
+    product: gravity
+    gravity:
+        version: 6.3.6
```

Comment:
What is this version here? Do we need to update this for every release?

Reply:
For Teleport we've started to use the specific binary version to make on-boarding smoother.  This makes it easier to update binary versions reference in the docs.. .e.g. https://gravitational.com/gravity/docs/quickstart/#getting-the-tools 

In Teleport we use it so we only have to update one place.
![image](https://user-images.githubusercontent.com/559288/76439584-d2f9cb80-6379-11ea-860a-b5606d110ee7.png)